chain-core/src/tx/witness/mod.rs

```diff
@@ -33,6 +33,23 @@ impl TxWitness {
     }
 }
 
+impl Encode for TxWitness {
```

Comment:
There is a `#[derive(Encode)]` on `TxWitness`. I'm not sure if it'll compile with both of them present.

Reply:
sorry wrong branch